Gemfile

```diff
@@ -52,6 +52,8 @@ gem 'turnout', '~> 2.4.0'
 gem 'uglifier', '~> 4.1.2'
 gem 'unicorn', '~> 5.4.0'
 gem 'whenever', '~> 0.10.0', require: false
+gem 'globalize', '~> 5.0.0'
+gem 'globalize-accessors'
```

Comment:
We need to fix this gem version to 0.2.1 with `gem 'globalize-accessors', '~> 0.2.1'` the same way every other gem in the Gemfile has